packages/office-addin-manifest/src/manifestHandler/manifestHandler.ts

```diff
@@ -7,21 +7,20 @@ import * as fs from "fs";
 
 export abstract class ManifestHandler {
   /* eslint-disable no-unused-vars */
-  constructor() {
-    this.manifestPath = "";
+  constructor(manifestPath: string) {
+    this.manifestPath = manifestPath;
     this.fileData = "";
   }
 
-  public async readFromManifestFile(manifestPath: string): Promise<string> {
-    this.manifestPath = manifestPath;
+  public async readFromManifestFile(): Promise<string> {
     try {
-      const fileData: string = await util.promisify(fs.readFile)(manifestPath, {
+      const fileData: string = await util.promisify(fs.readFile)(this.manifestPath, {
```

Comment:
Is there a reason we are reading the fileData and then the TeamsAppManifest objects in separate steps?

Reply:
I thought it would be needed previously. I just changed the code structure so we now just use the teams read package.